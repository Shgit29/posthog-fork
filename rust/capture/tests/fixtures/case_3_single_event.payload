{
  "event": "$pageview",
  "api_key": "phc_123abc456def",
  "timestamp": "2025-07-07T08:00Z",
  "distinct_id": "0197e2de-7996-74b8-845f-f423c1827f58",
  "properties": {
    "$os": "Android",
    "$browser": "Chrome",
    "$device": "Android",
    "$device_type": "Mobile",
    "$current_url": "https://example.com/the/page/?param=1",
    "$host": "www.example.com",
    "$pathname": "/the/page",
    "$browser_version": 74,
    "$browser_language": "en-US",
    "$screen_height": 720,
    "$screen_width": 1280,
    "$viewport_height": 3360,
    "$viewport_width": 1679,
    "$lib": "web",
    "$lib_version": "1.72.1",
    "$insert_id": "ghi567",
    "$time": 4575597.128,
    "$device_id": "123-abc-456-def",
    "$search_engine": "google",
    "utm_source": "google-ads",
    "utm_medium": "display",
    "utm_campaign": "foo-bar",
    "utm_content": "foobar",
    "utm_term": "foobar",
    "gclid": "abc123",
    "$referrer": "https://example.com/",
    "$referring_domain": "example.com",
    "title": "Hello Capture",
    "$session_id": "110ba456c70acf87-0781a83b47054d-72326443-e1000-110ba123d70af697",
    "$window_id": "110ba456d70afe68-08a3c2c5f5ee3d-72326443-e1000-110ba123e70b2190",
    "$pageview_id": "110ba456370bf098-0c284fe6144517-72326443-e1000-110ba124470c1b8f",
    "$set": {
      "utm_source": "google-ads",
      "utm_medium": "display",
      "utm_campaign": "xyz987",
      "utm_content": "foo-bar",
      "utm_term": "foobar",
      "gclid": "abc123"
    },
    "$set_once": {
      "$initial_os": "Android",
      "$initial_browser": "Chrome",
      "$initial_device_type": "Mobile",
      "$initial_current_url": "https://example.com/the/path/?param=1",
      "$initial_pathname": "/the/path",
      "$initial_browser_version": 74,
      "$initial_utm_source": "google-ads",
      "$initial_utm_medium": "display",
      "$initial_utm_campaign": "xyz987",
      "$initial_utm_content": "foo-bar",
      "$initial_utm_term": "foobar",
      "$initial_gclid": "abc123",
      "$initial_referrer": "https://example.com/",
      "$initial_referring_domain": "example.com"
    }
  }
}