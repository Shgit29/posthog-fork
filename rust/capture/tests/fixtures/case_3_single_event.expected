# TODO(eli): add JSON ProcessedEvent shape here!
